packages/solid/components/FocusRing/FocusRing.types.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2024 Comcast Cable Communications Management, LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import type { UIComponentProps } from 'types/interfaces.js';
+//import type { Signal } from 'solid-js';
+import type { ComponentStyleConfig, NodeStyleSet, Tone } from 'types/types.js';
+import type { NodeStyles } from '@lightningtv/solid';
+
+export interface FocusRingProps extends UIComponentProps {
+  /**
+   * placeholder for FocusRingProps
+   */
+}
```

Comment:
we can leave this for now, but if we wind up without any FocusRing specific props we should remove it